pkg/manager/member/pvc_resizer.go

```diff
@@ -226,6 +226,11 @@ func (p *pvcResizer) isVolumeExpansionSupported(storageClassName string) (bool,
 
 // patchPVCs patches PVCs filtered by selector and prefix.
 func (p *pvcResizer) patchPVCs(ns string, selector labels.Selector, pvcQuantityInSpec map[string]resource.Quantity) error {
+	if p.deps.StorageClassLister == nil {
```

Comment:
What about we just skip the expansion check here https://github.com/pingcap/tidb-operator/blob/master/pkg/manager/member/pvc_resizer.go#L260-L267, so that if the sc supports expansion, they can still resize the PVC and just return an error after the patch as we do now if not support.

Reply:
👍 fix in https://github.com/pingcap/tidb-operator/pull/3896/commits/956bf6118808e4ed6e4c559b3aa6d10d0c4e1634